### DISCUSSION TITLE
Left PFD freezing

### DISCUSSION THREAD

**Discussion by mickmcp** [flybywiresim/aircraft#3775] [Help] [answered]
Does anyone else get this, I'm using the development build but the stable one does it as well. A few minutes into the flight the PFD freezes, it works again if the wheels are lowered but freezes again if they are lifted back up. Co pilots always works fine.

Is this what they mean in the known bugswhen they say the PFD stops working?

**Comment by hotshotp** [comment, marked as answer]
known issue and will be fixed soon
